products/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=45, unique=True)
+    
+    class Meta:
+        db_table = 'categories'
+
+class Product(models.Model):
+    name          = models.CharField(max_length=45, unique=True)
+    price         = models.DecimalField(max_digits=8, decimal_places=2)
+    description   = models.CharField(max_length=255)
+    discount_rate = models.DecimalField(max_digits=3, decimal_places=2)
+    thumbnail_url = models.URLField(max_length=500)
```

Comment:
```suggestion
    thumbnail_url = models.URLField(max_length=2000)
```
이미지 유알엘 정보는 길이가 길수도 있기때문에 500자도 적습니다. 최소 2000자 주세요 아래 다른 URLFiled도 수정부탁드립니다.